src/Explorer/Panes/PanelManager.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import Explorer from "../Explorer";
+import { DeleteCollectionConfirmationPaneComponent } from "./DeleteCollectionConfirmationPaneComponent";
+
+export interface PanelParams {
+  headerText: string;
+  isOpen: boolean;
+  panelContent: JSX.Element;
+  onPanelClose: () => void;
+}
+
+export interface PanelManagerParams {
+  setPanelParams: (panelParams: PanelParams) => void;
+  setIsNotificationConsoleExpanded: (isExpanded: boolean) => void;
+}
+
+export class PanelManager {
```

Comment:
Lets sync on this offline. I want to talk about how this can be done with hooks.

Reply:
Discussed with Steve offline. I will remove PanelManager and pass panel state hooks to Explorer directly.